api4/command.go

```diff
@@ -51,8 +51,8 @@ func createCommand(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 
 	auditRec.Success()
-	auditRec.AddMeta("command_id", rcmd.Id)
 	c.LogAudit("success")
```

Comment:
Not specific for this change, but on what basis do we log a "success" line? Isn't it enough with the `auditRec.Success()`?

Reply:
Understood, thanks for the explanation!